cgmanifests/submodules/cgmanifest.json

```diff
@@ -60,9 +82,9 @@
       "Component": {
         "Type": "other",
         "other": {
-          "Name": "libxcrypt",
-          "Version": "4.4.17",
-          "DownloadUrl": "https://github.com/besser82/libxcrypt/archive/v4.4.17.tar.gz"
+          "Name": "swig",
```

Comment:
Looks like we're introducing a new dep. Should we add this to TPN file as well?

Reply:
It is a build time dependency, we don't distribute it.

Reply:
Ok, it wasn't clear from the comment if it was a build time dep. Thinking from new comers' point of view.